presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -449,145 +441,30 @@ private static boolean isValidPartitionType(com.facebook.presto.common.type.Type
                 REAL.equals(type) ||
                 DOUBLE.equals(type) ||
                 DATE.equals(type) ||
-                TIMESTAMP.equals(type) ||
+                type instanceof TimestampType ||
+                (TIME.equals(type) && fileFormat == PARQUET) ||
```

Comment:
Hmm I just noticed this. Do we not support time with ORC? or does ORC not have a time type?

Reply:
Orc do not have a time type. Referring to https://orc.apache.org/docs/types.html

And we do not support time type for Iceberg on Orc either.